test/support/utils_sycl.h

```diff
@@ -32,6 +32,7 @@
 #include "utils_invoke.h"
 #include "utils_test_base.h"
 
+#define TEST_USE_PREDEFINED_POLICIES (!defined(ONEDPL_USE_PREDEFINED_POLICIES) || ONEDPL_USE_PREDEFINED_POLICIES)
```

Comment:
Although this works, it produces a warning:
```
test/support/utils_sycl.h:100:9: warning: macro expansion producing 'defined' has undefined behavior [-Wexpansion-to-defined]
  100 | #    if TEST_USE_PREDEFINED_POLICIES
```
The following does not give a warning and I believe is semantically identical:
```suggestion
#ifdef ONEDPL_USE_PREDEFINED_POLICIES
#  define TEST_USE_PREDEFINED_POLICIES ONEDPL_USE_PREDEFINED_POLICIES
#else
#  define TEST_USE_PREDEFINED_POLICIES 1
#endif
```